app/src/main/java/io/mrarm/irc/util/IgnoreListMessageFilter.java

```diff
@@ -13,20 +18,43 @@
 
     public IgnoreListMessageFilter(ServerConfigData server) {
         mConfig = server;
+        mConfig.badCount = checkCompileIgnoreListEntries();
+    }
+
+    private Integer checkCompileIgnoreListEntries() {
```

Comment:
should return `int`; i do not think it needs to return anything if the warning toast is removed though?